docs/en/observability/ingest-logs-metrics-uptime.asciidoc

```diff
@@ -54,180 +45,244 @@ include::{tab-widgets}/add-fleet-server/widget.asciidoc[]
 
-Add the Elastic Synthetics integration to the default policy used by your agent.
-You use policies to manage settings across a group of agents. An agent policy may
-contain any number of integrations for collecting observability data from the various
-services running on your host.
+Add the Elastic Synthetics integration to your agent policy. You use policies to
+manage settings across a group of agents. An agent policy may contain any number
+of integrations for collecting observability data from the various services
+running on your host.
 
-. In {kib}, select **Add integrations**,
-and search for the **Elastic Synthetics** integration.
-+
-[role="screenshot"]
-image::images/kibana-fleet-integrations-synthetics.png[{fleet} showing Synthetics integration]
-. Click the Elastic Synthetics integration to see more details about it, then click
-**Add Elastic Synthetics**.
-+
-[role="screenshot"]
-image::images/kibana-fleet-integrations-synthetics-overview.png[{fleet} showing Synthetics integration overview]
+. In {kib}, go to the **Integrations** page (click **Add integrations** in the
+home page or main menu).
+
+. In the query bar, search for **Elastic Synthetics** and select the integration
+to see more details about it.
+
+. Click **Add Elastic Synthetics**.
 
-. On the **Add Elastic Synthetics integration** page, under **Configure integration**,
-enter the integration name and select your desired monitor type from the following monitor types:
+. Configure the integration name and select your desired monitor type from the
+following monitor types:
 +
 |===
 
-| *HTTP* | Connects via HTTP and verifies that the host returns the expected response.
+| **HTTP** | Connects via HTTP and verifies that the host returns the expected response.
 
-For detailed information
-about HTTP options, refer to our {heartbeat-ref}/monitor-http-options.html[Heartbeat] documentation.
+For detailed information about HTTP options, refer to our
+{heartbeat-ref}/monitor-http-options.html[Heartbeat] documentation.
 
-| *TCP* | Connects via TCP and verifies the endpoint by sending and receiving a custom payload.
+| **TCP** | Connects via TCP and verifies the endpoint by sending and receiving a custom payload.
 By default, the hostname and port are required.
 
 For detailed information about TCP options, refer to our
 {heartbeat-ref}/monitor-tcp-options.html[Heartbeat] documentation.
 
-| *ICMP* | Uses an ICMP `v4` and `v6` Echo Request to ping the configured hosts. By default,
+| **ICMP** | Uses an ICMP `v4` and `v6` Echo Request to ping the configured hosts. By default,
 the host name is required.
 
 For detailed information about ICMP options, refer to our
 {heartbeat-ref}/monitor-icmp-options.html[Heartbeat] documentation.
 
-| *Browser* | Runs automated tests using a real Chromium browser via the synthetics agent.
+| **Browser** a| Runs automated tests using a real Chromium browser via the synthetics agent.
```

Comment:
It's required to render embedded asciidoc. I've set it on this row rather than the table because asciidoc isn't embedded anywhere except where we have the NOTE.